client/signup/steps/import-preview/index.jsx

```diff
@@ -0,0 +1,209 @@
+					translate( 'Images' ),
+					translate( 'Comments' ),
+					translate( 'Tags' ),
+				],
+				unsupported: translate(
+					"We're unable to import your theme, plugins, comments, or online store."
+				),
+			},
+			'godaddy-gocentral': {
+				supported: [
+					translate( 'Pages' ),
+					translate( 'Blog posts' ),
+					translate( 'Images' ),
+					translate( 'Other media' ),
+				],
+				unsupported: translate(
+					"We're unable to import your online store, site layout, or styling."
+				),
+			},
+			medium: {
+				supported: [
+					translate( 'Posts' ),
+					translate( 'Images' ),
+					translate( 'Other media' ),
+					translate( 'Tags' ),
+				],
+				unsupported: translate( "We're unable to import your drafts, comments or followers." ),
+			},
+			squarespace: {
+				supported: [
+					translate( 'Posts' ),
+					translate( 'Pages' ),
+					translate( 'Images' ),
+					translate( 'Comments' ),
+					translate( 'Tags' ),
+				],
+				unsupported: translate(
+					"We're unable to import your online store, template, layout or styling."
+				),
+			},
+			wix: {
+				supported: [
+					translate( 'Pages' ),
+					translate( 'Blog posts' ),
+					translate( 'Images' ),
+					translate( 'Other media' ),
+				],
+				unsupported: translate(
+					"We're unable to import your online store, site layout, or styling."
+				),
+			},
+		};
+	}
+
+	componentDidMount() {
+		this.props.saveSignupStep( { stepName: this.props.stepName } );
+	}
+
+	submitImportPreview() {
+		this.props.submitSignupStep( { stepName: this.props.stepName } );
+		this.props.goToNextStep();
+	}
+
+	renderIcon() {
+		const {
+			signupDependencies: { importSiteEngine },
+		} = this.props;
+
+		const importerData = importerConfig()[ importSiteEngine ];
+
+		return (
+			<ImporterLogo
+				icon={ importerData.icon }
+				className={ 'import-preview__icon importer__service-icon ' + importSiteEngine }
+			/>
+		);
+	}
+
+	renderSupported() {
+		const {
+			signupDependencies: { importSiteEngine },
+		} = this.props;
+
+		// The preceding steps always provide a value for importSiteEngine
+		const items = this.previewData[ importSiteEngine ].supported || [];
+
+		const listItems = items.map( item => {
+			return (
+				<li className="import-preview__supported-item" key={ item }>
+					<Gridicon icon="checkmark" size={ 18 } className="import-preview__checkmark" />
+					<div className="import-preview__supported-text">{ item }</div>
+				</li>
+			);
+		} );
+
+		return <ul className="import-preview__supported">{ listItems }</ul>;
+	}
+
+	renderPreview() {
+		const {
+			signupDependencies: { importSiteEngine, importSiteUrl, siteTitle },
+			translate,
+		} = this.props;
+
+		let cardTitle = siteTitle;
+		let trimmedSiteUrl = '';
+
+		if ( isEmpty( importSiteUrl ) ) {
+			// The user has an import file and told us the source
+			const importerData = importerConfig()[ importSiteEngine ];
+			cardTitle = translate( 'Importing from ' ) + importerData.title;
```

Comment:
How about something like this?

```suggestion
			cardTitle = translate( 'Importing from %(importerName)s', {
				args: { importerName: importerData.title },
			} );
```

This makes the string more translatable in languages where the translation would need to put the engine name first.

Reply:
Ahh much better, thanks!